src/RichText/UserMention.php

```diff
@@ -190,8 +190,11 @@ public static function getUserIdsFromUserMentions(string $content)
 
         try {
             $content = Sanitizer::getVerbatimValue($content);
+            $content = htmlentities($content, ENT_QUOTES, 'UTF-8', false);
```

Comment:
Could you add a test case for the string you are trying to handle here? Rencoding the content will probably fail as `<span data-user-mention="true">...</span>` will probably be converted to `&lt;span data-user-mention=&quot;true&quot;&gt;...&lt;/span&gt;` and would not be parsable using xpath.

Reply:
This was to try to fix existing tests, including `Ticket::testScreenshotConvertedIntoDocument`, where there is a & in the content and a warning is shown saying there is no `;` in the entity.

I guess it could be ignored since it is only a warning and just move the `libxml_use_internal_errors` before the DOMDocument loads the HTML.